# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavutil/moz.build
# Commit: 13edc1b203a7
# Full Hash: 13edc1b203a76bf9d3bc00ba19a7f3b167a2ac45
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Directly use tx instead of the older avfft interface. r=karlt,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202156
# ==============================================================================

diff -r 71824d88e1cd -r 13edc1b203a7 media/ffvpx/libavutil/moz.build
--- a/media/ffvpx/libavutil/moz.build	Mon Feb 26 16:07:08 2024 +0000
+++ b/media/ffvpx/libavutil/moz.build	Mon Feb 26 16:07:09 2024 +0000
@@ -75,6 +75,10 @@
         ]
         USE_LIBS += ['mozva']
 
+EXPORTS.ffvpx = [
+    "tx.h"
+]
+
 SYMBOLS_FILE =  'avutil.symbols'
 NoVisibilityFlags()
 
